audio_service_platform_interface/lib/utils.dart

```diff
@@ -0,0 +1,4 @@
+part of 'package:audio_service_platform_interface/audio_service_platform_interface.dart';
+
+@pragma('vm:prefer-inline')
+Map<String, dynamic>? _castMap(Map? map) => map?.cast<String, dynamic>();
```

Comment:
we want to inline this

Reply:
Maybe all uses of this can go away via `invokeMap`?